src/skate.js

```diff
@@ -84,7 +84,7 @@ function skate (id, definition) {
   definition = inherit(definition || {}, skate.defaults);
   definition.id = id;
   if (definition.prototype === skate.defaults.prototype) {
-    definition.prototype = {};
+    definition.prototype = Object.create(HTMLElement.prototype);
```

Comment:
Rather than only setting this if the caller didn't provide a `prototype` in the `definition`, perhaps we should insert `HTMLElement.prototype` into `definition.prototype`'s prototype chain.

Doing so may make less existing code break.